Consul/Client.cs

```diff
@@ -443,8 +443,6 @@ public ConsulClient(ConsulClientConfiguration config)
         /// </summary>
         /// <param name="config">A Consul client configuration</param>
         /// <param name="client">A custom HttpClient</param>
```

Comment:
It would be nice to add a comment, that it is responsibility of the user to handle the lifecycle of `httpclient`.

Reply:
Great suggestion @marcin-krystianc I updated the documentation for this parameter.